.github/workflows/release-please.yml

```diff
@@ -0,0 +1,31 @@
+# .github/workflows/release-please.yml
+
+name: Release Please
```

Comment:
I don't think this workflow is required at all since we moved to a bot. See https://github.com/microsoftgraph/msgraph-beta-sdk-java/pull/952